samples/ABNF/tranquil.abnf

```diff
@@ -0,0 +1,61 @@
+; Source:  https://github.com/fjolnir/Tranquil
+; License: BSD 3 Clause License
+; Modified for standard compatibility by René Schwaiger
```

Comment:
[Here's the relevant rule](https://github.com/sanssecours/ABNF.tmbundle/blob/master/Syntaxes/ABNF.tmLanguage#L552-L568) which I believe is applying the error highlighting:

```
[a-zA-Z][a-zA-Z0-9\-]*|(?<invalid>\S)
```

So it's basically saying "highlight anything that's not whitespace, and that isn't an ASCII word character or dash that commences with an alphabetic ASCII character.